Makefile

```diff
@@ -39,7 +39,7 @@ BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 # Image URL to use all building/pushing image targets
 IMG ?= controller:latest
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
-CRD_OPTIONS ?= "crd:trivialVersions=true"
+CRD_OPTIONS ?= "crd:crdVersions={v1},trivialVersions=true"
```

Comment:
can the CRD version bump be in a separate commit in this PR ? or a separate PR ?

i.e recreate CRDs with the new operator SDK without switchdev change

Reply:
Addressed by https://github.com/openshift/sriov-network-operator/pull/389